src/systempolicy/systemPolicy.go

```diff
@@ -1279,7 +1279,7 @@ func GenFileSetForAllPodsInCluster(clusterName string, pods []types.Pod, settype
 		} else {
 			if !reflect.DeepEqual(mergedfs, out[wpfs]) {
 				log.Info().Msgf("updating wpfs db entry for wpfs=%+v", wpfs)
-				err = libs.UpdateWorkloadProcessFileSetMySQL(CfgDB, wpfs, mergedfs)
+				err = libs.UpdateWorkloadProcessFileSetSQLite(CfgDB, wpfs, mergedfs)
```

Comment:
This will not work when the driver is set to mysql. Note that we also have to support mysql.
CC @seswarrajan